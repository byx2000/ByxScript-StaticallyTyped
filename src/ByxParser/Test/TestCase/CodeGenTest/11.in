function void f()
{
	return;
}

function void main()
{

}

function void g()
{
	return;
	return;
}

