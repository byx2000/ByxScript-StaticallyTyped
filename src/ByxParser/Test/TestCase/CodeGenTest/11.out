3
0 2
0 0
0 3

0

ret
ret
ret
ret
ret
ret